components/Button/__tests__/Button.test.js

```diff
@@ -57,13 +57,13 @@ describe('Button', () => {
     expect(wrapper.prop(attribute)).toBeUndefined();
   });
 
-  it('should send log to console when clickHandler is called in non-prod environment', () => {
+  it('should send log to console when button is clicked in non-prod environment', () => {
     /* eslint-disable no-console */
     console.log = jest.fn();
 
     const ButtonShallowInstance = shallow(<Button>Testing</Button>);
 
-    ButtonShallowInstance.instance().clickHandler();
+    ButtonShallowInstance.simulate('click');
```

Comment:
This now tests the functionality, not the implementation.

Reply:
Yep yep! Thanks for this. Excited for #850 cuz it'll automatically resolve a bunch of these.